demos/_unit-test/scope-builder.php

```diff
@@ -43,11 +48,10 @@
 });
 
 $expectedWord = <<<'EOF'
-     Project Budget is greater or equal to '1000' 
-     and (Project Name is regular expression '[a-zA-Z]' 
-            and Client Country Iso is equal to 'Brazil' 
-            and Start Date is equal to '2020-10-22') 
-    and (Finish Time is not equal to '22:22' or Is Commercial is equal to '0')
+    Project Budget is greater or equal to '1000' 
+    and (Project Name is regular expression '[a-zA-Z]' 
+    and Client Country Iso is equal to 'Brazil' and Start Date is equal to '2020-10-22') 
```

Comment:
Should be "Client Country Iso is equal to 'BR'" here isn't it?

Reply:
Change should be done in Model/Scope::toWords() method in atk4/Data